scripts/go/configs/.golangci.toml

```diff
@@ -44,10 +44,6 @@ enable = [
 # "gocyclo",
 # "unparam"
 
-[[issues.exclude-rules]]
-path = "_test\\.go"
-linters = ["scopelint"]
-
```

Comment:
Why this is removed? Is that intentional?

Reply:
@papagian Yes it's intentional, because scopelint gets replaced with exportloopref, and the latter doesn't trigger on our tests (hopefully it's just more accurate).